plugins/modules/mount.py

```diff
@@ -296,9 +299,9 @@ def umount(module):
     if mount_dir or mount_over_dir:
         if is_fspath_mounted(module, mount_dir, mount_over_dir) is False:
             if mount_dir:
-                result['msg'] = "Filesystem/Mount point '%s' already mounted" % mount_dir
+                result['msg'] = "Filesystem/Mount point '%s' already unmounted" % mount_dir
```

Comment:
Just my opinion, but I think the message here would be more clear if it was:
"Filesystem/Mount point is not mounted"
because if I try to umount something that isn't mounted on an aix machine, it says "cannot find anything to unmount" and it could be because nothing was successfully mounted in the first place. It could be misleading to give an error message that implies something was mounted and unmounted. I hope that makes sense.

Reply:
it does sound better. okay i will chang the msg to
"Filesystem/Mount point '%s' is not mounted"

Reply:
i have changed the msg